tuna/connection.py

```diff
@@ -132,18 +148,25 @@ def is_alive(self):
         self.logger.error('Checking isAlive when channel does not exist')
       return not self.out_channel.exit_status_ready()
 
-  def exec_command_unparsed(self, cmd, timeout=SSH_TIMEOUT, abort=None):
+  def exec_command_unparsed(self, cmd: str, timeout: int =int(SSH_TIMEOUT),\
+  abort:Union[Any, None]= None) -> Tuple[Any, Any, Any]:
```

Comment:
@alexandraBara can you comment on typing for abort? It looks like we are using it as a bool, but I haven't found it set at any point.

Reply:
Returns are file-like objects, first may be None because we don't capture stdin for Popen.

Reply:
It looks like a bool correct, i also dont see it set differently anywhere.